lib/pagination.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Pagination
+  def paginated_result(cursor, page_size, order_column)
```

Comment:
Could you provide more details about our pagination logic? I understand you've already included some information in our internal documentation, but we also use commit messages as historical records. It would be beneficial to include the primary rationale behind this logic in the commit message. Over time, this might be modified or improved. Knowing the design considerations at the time of the code commit is useful.